### ISSUE TITLE
No "Log out"/"Delete account" options right after registering until the menu is refreshed

### ISSUE THREAD

**Issue by yhm4ps1jfp67nkv** [libre-tube/LibreTube#1569]
### Steps to reproduce

1. Open the Instance settings
2. Tap "Log in/register"
3. Enter the username and password
4. Click "Register"

### Expected behavior

"Log out" and "Delete account" options should be shown right away

### Actual behavior

"Log out" and "Delete account" options are shown only after reopening the menu

### LibreTube version

Latest Debug

### Android version

Android 12

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.
